server/rules.go

```diff
@@ -168,6 +169,7 @@ func (server HTTPServer) getFeedbackAndTogglesOnRule(
 		rule.UserVote = types.UserVoteNone
```

Comment:
same here, wrong indentation?

Reply:
it looks ok in my ide, probably just line breaks on github?